docker/candlepin-base-cs9/setup-devel-env.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+#
+# Sets a system up for a candlepin development environment (minus a db,
+# handled separately), and an initial clone of candlepin.
+
+set -ve
+
+source /root/dockerlib.sh
+
+export JAVA_VERSION=11
+export JAVA_HOME=/usr/lib/jvm/java-$JAVA_VERSION
+
+# Install & configure dev environment
+dnf install -y epel-release
+
+PACKAGES=(
+    createrepo_c
+    expect
+    gettext
+    git
+    hostname
+    java-$JAVA_VERSION-openjdk-devel
+    jss-5.0.3-1.el9 # last jss package that support Java version 11
+    mariadb
+    openssl
+    pki-servlet-engine
+    postgresql
+    procps
+    python3-pip
+    python3-pyyaml
+    python3-requests
+    python3-libxml2
+    python-unversioned-command
+    rpm-build
+    rpm-sign
+    rsyslog
+    wget
+    which
+    zlib
+    zlib-devel
```

Comment:
I think these 2 were only dependencies for ruby/rspec. Can you test if everything works if you remove them from the image?

Reply:
I tried it both with postgres and mysql and it works in both cases without the two packages